superset-frontend/cypress-base/cypress/integration/dashboard/load.test.ts

```diff
@@ -23,12 +23,18 @@ import {
 } from './dashboard.helper';
 
 describe('Dashboard load', () => {
-  before(() => {
+  beforeEach(() => {
```

Comment:
Should this be a `beforeAll` so we're not doing unnecessary repeat logins?

Reply:
I think I tried that and it broke 🤷‍♂️